HW31/task2.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+user_input = input("Input your city: ")
+
+res = requests.get(f"https://geocoding-api.open-meteo.com/v1/search?name={user_input}")
+try:
```

Comment:
nit: better to use several functions to structure the code like:
get_city_coordinates
get_current_weather
show_temperature